### PR TITLE
docs: add @wg-ecosystem meeting notes 2020-10-29

### PR DESCRIPTION
**Pull request by erickzhao**

### CODE REVIEW

**Comment by erickzhao** [comment]
@vhashimotoo I forgot what the problem was for https://github.com/electron/electronjs.org/pull/4319. Is the initial auto-update problem fixed?

**Comment by vhashimotoo** [comment]
> @vhashimotoo I forgot what the problem was for electron/fiddle#516. Is the initial auto-update problem fixed?

@erickzhao The problem here that we generate and upload `nupkg` file for a 32-bit system for auto-updating, this not resolved and I'm still not sure how to fix them. This PR in that state covers migrating to Forge publisher, but not fixing auto-updating.

File: wg-ecosystem/meeting-notes/2020-10-29.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+- @vhashimotoo stepping down from Electron governance.
+  - Vlad is leaving his role in the working group and resigning as chair. He won't be available on Slack anymore,
+  but will still be active on the Electron Discord server (and will still maintain his admin role there).
+  - Work to hand over:
+    - [electron-i18n#785](https://github.com/electron/i18n/pull/785): Support for the docs to handle multiple major versions of Electron.
+    This PR works, but build times increase massively (from ~5 minutes to ~40 minutes). His suggested alter
+    - [electronjs.org#4319](https://github.com/electron/electronjs.org/pull/4319): Migrating electronjs.org from Browserify to Webpack.
+    This PR only builds the JS files, so work needs to be done to compile all of our SCSS.
+    - [fiddle#516](https://github.com/electron/electronjs.org/pull/4319): Migrating Fiddle's publishing config from `softprops/action-gh-release`
```
Comment (vhashimotoo) [inline wg-ecosystem/meeting-notes/2020-10-29.md:22]:
```suggestion
    - [fiddle#516](https://github.com/electron/fiddle/pull/516): Migrating Fiddle's publishing config from `softprops/action-gh-release`
```

**Review by vhashimotoo — APPROVED** [review]

**Review by malept — APPROVED** [review]

File: wg-ecosystem/meeting-notes/2020-10-29.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* @vhashimotoo
+
+## Agenda
+
+- @vhashimotoo stepping down from Electron governance.
+  - Vlad is leaving his role in the working group and resigning as chair. He won't be available on Slack anymore,
+  but will still be active on the Electron Discord server (and will still maintain his admin role there).
+  - Work to hand over:
+    - [electron-i18n#785](https://github.com/electron/i18n/pull/785): Support for the docs to handle multiple major versions of Electron.
+    This PR works, but build times increase massively (from ~5 minutes to ~40 minutes). His suggested alter
```
Comment (malept) [inline wg-ecosystem/meeting-notes/2020-10-29.md:19]: This sentence trails off :smile: 

```suggestion
    This PR works, but build times increase massively (from ~5 minutes to ~40 minutes). His suggested alter
```

**Review by malept — COMMENTED** [review]
I missed something

File: wg-ecosystem/meeting-notes/2020-10-29.md (hunk 2 above)
Comment (erickzhao) [reply wg-ecosystem/meeting-notes/2020-10-29.md:19]: Good point, I'll fix that. The alternative IIRC was to do the multi-version build on the website's side I believe.

**Pull request merged**